src/components/breadcrumbs/crumb/crumb.component.tsx

```diff
@@ -24,26 +24,52 @@ export interface CrumbProps
       | "disabled"
     >,
     TagProps {
-  /** This sets the Crumb to current, does not render Link */
   isCurrent?: boolean;
+  onClick?: (
```

Comment:
comment: the interface already extends `LinkProps` which has `onClick` defined so you don't need this it can just be destructured like you have below